docs/anchoring-assets/anchor-platform/sep24/integration.mdx

```diff
@@ -464,6 +470,24 @@ Finally, after the transaction has been successfully sent and submitted to the n
 
 </CodeExample>
 
+## Extra Transaction Updates
+
+### Expiring Transactions
+
+Your business may want to expire transactions that have been abandoned by the user after some time. It's a good practice to clean up transactions in `inactive` status. To do so, simply change the transaction status to `expired`
```

Comment:
We don't have an `inactive` status so I don't think we need that sentence.

Reply:
Yeah it has wrong wording, let me fix that